devtools/pixi-envs/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[project]
+name = "openff-utilities"
+dependencies = [ "mypy==1.6", "pytest-cov>=5.0.0,<6", "types-setuptools>=75.2.0.20241019,<76", "types-toml>=0.10.8.20240310,<0.11", "pytest", "importlib-resources"]
```

Comment:
The version requirements here are fairly strict. I'm probably going to forget to update these manually - can these get automatically updated, or perhaps could some of these upper constraints be removed?

Reply:
I think we should probably be okay with removing some of the upper bounds if you would prefer that. You want me to change them and make them only major or minor bound?

Reply:
On a closer look, this probably looks so out of date to me because of this pin, which is my responsibility to update: https://github.com/openforcefield/openff-utilities/blob/98acba9ae171c9c78e7a5bf001559658fde88082/devtools/conda-envs/test_env.yaml#L13

Since this works fine, just leave it be and I'll try to update it alongside a version of `mypy` that's from this year